src/main/java/com/github/onlinebookstore/model/User.java

```diff
@@ -24,7 +25,8 @@
 @Entity
 @Getter
 @Setter
-@RequiredArgsConstructor
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
Remove
@AllArgsConstructor
@NoArgsConstructor

Aslo, add constraints where is needed. For example email should be unique

Reply:
Not fixed.